adapters/adgeneration/adgenerationtest/exemplary/single-banner.json

```diff
@@ -73,7 +73,8 @@
                     "X-Forwarded-For": [
                         "0.0.0.0"
                     ]
-                }
+                },
```

Comment:
It is a shame that adgeneration does not have any multi-imp tests. But that is beyond the scope of this PR. We will have to try to be more picky about that in adapter reviews.